docs/dev/projectsetup/scala_api_quickstart.md

```diff
@@ -217,7 +217,7 @@ take a look at the [Stream Processing Application Tutorial]({{ site.baseurl }}/g
 If you are writing a batch processing application and you are looking for inspiration what to write,
 take a look at the [Batch Application Examples]({{ site.baseurl }}/dev/batch/examples.html)
 
-For a complete overview over the APIa, have a look at the
+For a complete overview over the API, have a look at the
```

Comment:
```suggestion
For a complete overview over the APIs, have a look at the
```
?

Reply:
I think it is just referring to one API so it doesn't need to be plural?

Reply:
We list 2 APIs in the following lines; so yes I would say so.